lang/cs/Org.Apache.REEF.Client/Local/LocalClient.cs

```diff
@@ -132,16 +134,23 @@ public IDriverHttpEndpoint SubmitAndGetDriverUrl(IJobSubmission jobSubmission)
                 ));
 
             var fileName = Path.Combine(driverFolder, _fileNames.DriverHttpEndpoint);
-            HttpClientHelper helper = new HttpClientHelper();
-            _driverUrl = helper.GetDriverUrlForLocalRuntime(fileName);
+            JobSubmissionResultImpl result = new JobSubmissionResultImpl(this, fileName, false);
 
-            Logger.Log(Level.Info, "Submitted the Driver for execution. Returned driverUrl is: " + _driverUrl);
-            return helper;
+            Logger.Log(Level.Info, string.Format(CultureInfo.CurrentCulture,
```

Comment:
Please pass the format string straight to the logger. That way, it won't be created unless needed.